packages/fx-core/resource/package.nls.json

```diff
@@ -305,7 +302,9 @@
   "core.createProjectQuestion.openAiPluginManifest.multipleValidationErrors.vscode.message": "Invalid OpenAI plugin manifest. Check [output panel](command:fx-extension.showOutputChannel) for details.",
   "core.createProjectQuestion.openAiPluginManifest.validationError.missingApiUrl": "Missing URL in \"%s\".",
   "core.createProjectQuestion.openAiPluginManifest.validationError.authNotSupported": "Auth type is not supported. Supported auth type: \"%s\".",
-  "core.createProjectQuestion.meArchitecture.title": "Architecture of Search Based Message Extension",
+  "core.createProjectQuestion.capability.apiMe.label": "API Based Search Message Extension",
+  "core.createProjectQuestion.capability.apiMe.detail": "Retrieve data from external systems using APIs",
+  "core.createProjectQuestion.apiMeQuestion.title": "Plugin for Microsoft Teams",
```

Comment:
Do we have this term?
Plugin got MS Teams? isn't it just API Based Search Message Extension for Teams?

Reply:
No, we shouldn't have this term. This PR is to **revert** a pr to merge from main to dev (because that pr has issue on merge strategy), so the diff will contain changes like this.

I will merge from main to dev again after this revert so the final result in dev branch will not contain this term.